newm-server/src/main/kotlin/io/newm/server/database/migration/V77__ReleasesUpdates.kt

```diff
@@ -0,0 +1,26 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V77__ReleasesUpdates : BaseJavaMigration() {
```

Comment:
Since, as per my comment, the other one should be `V77`, should this one become `V78`?